cli/cmd/plugin/ui/web/tanzu-ui/src/shared/constants/NavRoutes.constants.tsx

```diff
@@ -11,7 +11,7 @@ export const NavRoutes = {
     // cluster create pages
     MANAGEMENT_CLUSTER_SELECT_PROVIDER: '/management-cluster-provider',
     WORKLOAD_CLUSTER_LANDING: '/workload-cluster-landing',
-
+    UMANAGED_CLUSTER_WIZARD: '/unmanaged-cluster-wizard',
```

Comment:
I'm wondering if this route is more the `/unmanaged-cluster-landing` page (since it's just displaying the unmanaged clusters) and you'd want to reserve `/unmanaged-cluster-wizard` for the workflow that creates an unmanaged cluster?

Reply:
Yes that route will be for the cluster wizard I am currently in the process of developing it. Should I move the route? I put it there since I saw the workload cluster wizard there.

Reply:
Yes this is correct to leave it as 'wizard' since this is for that wizard route